plugins/nginx/nginx_byprojects/README.md

```diff
@@ -6,41 +6,35 @@ Count the number of hits per projects/vhost.
 ![byproject_access](https://www.mantor.org/~northox/misc/munin-plugins/nginx_byprojects_access1-month.png "byproject_access")
 
 ## munin_byprojects_bandwidth
-Count the total bandwidth used by each projects/vhost. [Logtail] (https://www.fourmilab.ch/webtools/logtail/) is required.  
+Count the total bandwidth used by each projects/vhost. [Logtail](https://www.fourmilab.ch/webtools/logtail/) is required.  
 ![byproject_bandwidth](https://www.mantor.org/~northox/misc/munin-plugins/apache_byprojects_bandwidth-month.png "byproject_bandwidth")
 
 ## munin_byprojects_inout_bandwidth
-Counts the in/out bandwidth used by each projects/vhost. [Logtail] (https://www.fourmilab.ch/webtools/logtail/) is required.  
+Counts the in/out bandwidth used by each projects/vhost. [Logtail](https://www.fourmilab.ch/webtools/logtail/) is required.  
 ![byproject_inout_bandwidth](https://www.mantor.org/~northox/misc/munin-plugins/apache_byprojects_inout_bandwidth-month.png "byproject_inout_bandwidth")
 
 ## Installation
 The setup is pretty straight forward. First you need to configure the plugin:
 
-Define the path to logtail:
+In your munin plugin configuration file (for example, a new dedicated /etc/munin/plugin-conf.d/nginx_byprojects), configure the plugins : 
 
-      $logtail = '/usr/local/bin/logtail';
+      [byprojects_*]
+      env.logtail /usr/local/bin/logtail
 
-Multiple logs can be used for the same project/vhost and a regular expression (regex) can be used as a filter:
+Multiple logs can be used for the same project/vhost and a regular expression (regex) can be used as a filter.
+Each log is defined in a dedicated environment variable, named env.site.[siteName]. The value is JSON formatted.
 
-      my %logs = (
-        'prod' => [
-                    {'path' => '/home/prod/log/access.log'}
-                  ],
-         'dev' => [
-                    {'path' => '/var/log/httpd/ssl-dev-access.log'},
-                    {'path' => '/home/dev/log/access*.log'} # glob is supported
-                  ],
-        'test' => [
-                    {'path' => '/var/log/access.log', 'regex' => '"[A-Z]+ /test/'},
-                    {'path' => '/home/test/log/access.log'}
-                  ],
-      );
+      env.site.prod 	[{"path":"/home/prod/log/access.log"}]
```

Comment:
Please use spaces if you really need to align the items - otherwise you cannot predict the visual representation for different tab sizes.